.github/workflows/finalize-release.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     name: Cherry-pick Changelogs
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           token: ${{ secrets.IMJS_ADMIN_GH_TOKEN }}
           ref: ${{ github.ref }} # checkouts the branch that triggered the workflow
```

Comment:
update the user.email on line 28,
we dont need the random numbers and the + in front
keep it consistent with the other pipelines
not often we review all these pipelines, lets try to make sure theyre all up to date and clean